bin/bank/pycbc_aligned_bank_cat

```diff
@@ -165,36 +165,13 @@ if options.metadata_file:
                                  contenthandler=LIGOLWContentHandler)
 else:
     outdoc = None
-if options.output_file.endswith(('.xml','.xml.gz','.xmlgz')):
-    llw_output.output_sngl_inspiral_table(
-        options.output_file,
-        temp_bank,
-        metricParams,
-        ethincaParams,
-        programName=__program__,
-        optDict=options.__dict__,
-        outdoc=outdoc
-    )
-elif options.output_file.endswith(('.h5','.hdf','.hdf5')):
-    out_fp = h5py.File(options.output_file, 'w')
-    out_fp['mass1'] = temp_bank[:,0]
-    out_fp['mass2'] = temp_bank[:,1]
-    out_fp['spin1z'] = temp_bank[:,2]
-    out_fp['spin2z'] = temp_bank[:,3]
-    out_fp['f_lower'] = [options.f_low] * len(temp_bank[:,0])
-    tmplt_durations = []
-    for i in range(len(temp_bank[:,0])):
-        gwflit = get_waveform_filter_length_in_time
```

Comment:
I have actually removed the ability to output `template_duration` and `approximant` here, but I was not sure that this is needed? It definitely isnt for some use cases, and it seemed complicated to add/not add in the general case

Reply:
I forget if they are *needed* (I suspect PyCBC Live needs one of them at least) but in any case I would not remove them, they *should* be in the bank!

Reply:
I have added the options back, and set them specifically for this code. It may be worth doing for all codes in future, but this MR is to fix a specific problem

Reply:
I think it's fine that duration should be set specifically for this code (or for the set of PyCBC metric bank codes). The metric implicitly uses SPATmplt (or TaylorF2, same thing), pycbc_brute_bank does not, and should therefore be responsible for setting it's own duration values.